### PR TITLE
Enabling flyway

### PR DESCRIPTION
**Pull request by burzum619**
Enabling flyway migrations for the exam service.

### CODE REVIEW

File: service/src/main/docker/docker-startup.sh [hunk 1]
```diff
@@ -19,6 +19,10 @@ java \
     -Dexam-service.student-url=http://student/ \
     -Dexam-service.assessment-url=http://assessment/ \
     -Dexam-service.config-url=http://config/ \
+    -Dflyway.enabled=true \
```
Comment (millerts) [inline service/src/main/docker/docker-startup.sh]: Do we want to have a separate shell script called docker-startup-development.sh that will contain these values?

Comment (burzum619) [inline service/src/main/docker/docker-startup.sh]: Or I can change this to ${FLYWAY_ENABLED} and pass this in from TC.

Comment (millerts) [inline service/src/main/docker/docker-startup.sh]: Sounds great. 👍

**Comment by millerts** [comment]
Approve after that one small tweak.

**Pull request merged**